scalafix/output/src/main/scala/fix/ZIOSpec.scala

```diff
@@ -11,7 +11,7 @@ import zio.test._
 
 import scala.annotation.tailrec
 import scala.util.{Failure, Success, Try}
-import zio.{ Clock, FiberId, Random, Random, _ }
+import zio.{ Clock, FiberId, Random, _ }
```

Comment:
Was imported twice

Reply:
Should this just be `import zio._` or `zio.{ Clock, FiberId, Random }`?

Reply:
No idea. I just remove an compilation issue when I'm compiling with Intellij